torch_np/_ndarray.py

```diff
@@ -157,7 +166,8 @@ def view(self, dtype):
         tview = self.tensor.view(torch_dtype)
         return ndarray(tview)
 
-    def fill(self, value):
+    @normalizer
+    def fill(self, value: ArrayLike):
         self.tensor.fill_(value)
```

Comment:
In PyTorch `fill_` just works with Python scalars, it doesn't accept ArrayLikes. For ArrayLikes you need to use `copy_` and perhaps perform some shape check?

Reply:
Ah, I see. Since I saw the `ArrayLike` annotation, I tought it referred to arbitrary input arrays.

Reply:
It could have been clearer to not have the annotation indeed, but numpy allows numpy scalars and 0d arrays, so here we are.

Reply:
Yep. Let's just add a short comment, but otherwise LGTM.